distribution/osbuild-composer-clouddot-template.yml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+kind: Template
+labels:
+  app: osbuild-composer
+  template: osbuild-composer
+metadata:
+  annotations:
+    description: OCP template for osbuild-composer in cloud.redhat.com
+  name: osbuild-composer
+objects:
+
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    labels:
+      service: osbuild-composer
+    name: osbuild-composer
+  spec:
+    replicas: 2
```

Comment:
I think we want just 1 replica because osbulid-composer without a database isn't prepared to have multiple replicas.